WordPress/src/main/java/org/wordpress/android/ui/pages/PagesFragment.kt

```diff
@@ -64,7 +63,6 @@ class PagesFragment : Fragment() {
     private lateinit var viewModel: PagesViewModel
     private lateinit var swipeToRefreshHelper: SwipeToRefreshHelper
     private lateinit var actionMenuItem: MenuItem
-    @Inject lateinit var postStore: PostStore
```

Comment:
are you sure this is not necessary? because usually we're injecting these "unused" classes because we need to initialize them in FluxC